README.md

```diff
@@ -187,12 +189,49 @@ The resolver presents a simple `resolver()` function that returns a ES6 Promise
 
 ```js
 import resolve from 'did-resolver'
-import registerResolver from 'ethr-did-resolver'
+import register from 'ethr-did-resolver'
 
-registerResolver()
+register()
 
 resolve('did:ethr:0xf3beac30c498d9e26865f34fcaa57dbb935b0d74').then(doc => console.log)
 
 // You can also use ES7 async/await syntax
 const doc = await resolve('did:ethr:0xf3beac30c498d9e26865f34fcaa57dbb935b0d74')
 ```
+
+## Resolving a DID document in a private chain
+
+The `register(conf)` function allows to configure diferent private networks.
+
+The `conf` object can have the following parameters:
+
+| Param                  | Description                                                                  |
+| ---------------------- | ---------------------------------------------------------------------------- |
+| `provider`             | web3 provider object for mainnet                                             |
+| `web3`                 | web3 object (will use web3.currentProvider) for mainnet                      |
+| `rpcUrl`               | JSON-RPC URL to access mainnet                                               |
+| `registry`             | Address of the registry on mainnet                                           |
+| `networks`             | array of non mainnet network and their configs                               |
+| `networks[0].name`     | name of the private network. Same as in `did:ethr:privateNetworkName:0x....` |
+| `networks[0].provider` | web3 provider object for the private network                                 |
+| `networks[0].web3`     | web3 object (will use web3.currentProvider) for the private network          |
+| `networks[0].rpcUrl`   | JSON-RPC URL to access the private network                                   |
+| `networks[0].registry` | Address of the registry on the private network                               |
+
+Either provider, web3 or rpcUrl is needed.
+
+```js
+import resolve from 'did-resolver'
+import register from 'ethr-did-resolver'
+
+const registyAddress='0x....' //Registry address on privChain
+const jsonRpcEndpoint='https://priv.chain/rpc' //JSON-RPC endpoint for privChain
```

Comment:
This configuration does not seem to explicitly support endpoints that require authentication.
Is there any guidance about how that should be handled?

Reply:
No support for authenticated endpoints in this PR